app/utils/Stats.scala

```diff
@@ -0,0 +1,19 @@
+package utils
+
+case class Percentiles(values: Seq[Int]) {
+
+  private def percentile(s: Seq[Int], p: Double): Int = {
```

Comment:
Strange choice to use a Double for a percentile! Why not Ints in the range [0, 100]?